lib/matplotlib/contour.py

```diff
@@ -1751,6 +1757,18 @@ def _initialize_x_y(self, z):
     iterable is shorter than the number of contour levels
     it will be repeated as necessary.
 
+negative_linestyles : None or str, optional
+                      {'solid', 'dashed', 'dashdot', 'dotted'}
```

Comment:
```suggestion
negative_linestyles : {*None*, 'solid', 'dashed', 'dashdot', 'dotted'}, \
optional
```
(or where ever the line should break.)

Reply:
I went ahead and made this exact change. It matches `linestyles` formatting... I'm not sure where I saw the formatting I initially had.